iModelCore/Units/PublicAPI/Units/UnitRegistry.h

```diff
@@ -26,173 +26,10 @@ friend struct Unit;
 
     void AddSystem(UnitSystemR unitSystem); // Used for creating the static definitions.
 
-    PhenomenonP AddPhenomenonInternal(Utf8CP phenomenaName, Utf8CP definition)

```

Comment:
I'd rather leave the implementation in the header ... I'm trying to remember why we did it but I remember we did it for some reason.